core/src/components/tab-button/test/shape/tab-button.e2e.ts

```diff
@@ -0,0 +1,55 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+import type { E2EPage, E2EPageOptions, ScreenshotFn } from '@utils/test/playwright';
+
+class TabButtonFixture {
```

Comment:
Why was a fixture created?

Reply:
This file was created based on toast/test/shape/toast.e2e.ts which also has a fixture and it made sense for me to have one here in order to access the same page and elements on every test.